benchmark/bench/EpiInfModels/Renewal.jl

```diff
@@ -0,0 +1,34 @@
+# Error:  ArgumentError: Converting an instance of ReverseDiff.TrackedReal{Float64, Float64, Nothing} to Float64 is not defined. Please use `ReverseDiff.value` instead.
```

Comment:
Are these auto-generated error messages added to the file by benchmarking or personal notes?

Reply:
Ah, these are noted in the opening message.

Reply:
There is an issue here as well #338 where we could also store these?